nima_core/storage/unified_db.py

```
@@ -81,6 +87,13 @@ def get_unified_conn(db_path: str | Path | None = None) -> Any:
 def _create_schema(conn: Any) -> None:
     """Create all tables if they don't exist."""
     conn.execute("PRAGMA foreign_keys = ON")
+
+    # Migration: Add workspace_path to existing memory_nodes table
+    try:
+        conn.execute("ALTER TABLE memory_nodes ADD COLUMN workspace_path TEXT")
+    except Exception:
+        pass  # Column already exists
```

Ruff (0.15.7)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)